.github/actions/build-android/action.yml

```
@@ -0,0 +1,74 @@
+# This is coded as an action rather than a workflow so the release-tools
+# repository can load it from a dynamically-chosen commit. Cross-repository
+# workflow calls must have a Git reference hard-coded in the calling workflow,
+# but actions can be run dynamically from the runner's filesystem.
+
+name: Build and test (Android)
+description: Build and test (Android)
+
+inputs:
+  triplet:
+    description: Host triplet
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    # The steps of a composite action are not clearly divided in the GitHub
+    # UI, so use a single step with ::group:: markers instead.
+    - shell: bash
+      env:
+        triplet: ${{ inputs.triplet }}
+      run: |
+        echo "::group::Configure build Python"
```

A minor cosmetic thing - GitHub Actions prints the first line of a multiline `run` literally as part of the console output - so maybe insert an `echo` so that the first output isn't a `::group::` line.
```suggestion
        echo "Build and test Android"
        echo "::group::Configure build Python"
```

---

An `echo` would cause the line to be duplicated in the collapsed view, so I've used a comment instead.